src/hooks/useSections.ts

```diff
@@ -41,19 +48,20 @@ export default function useSections(
   // Fetch Recommendations Results
   const recommendationsResults = useFetchRecommendationPod(cioClient, recommendationsSections);
 
-  const sectionResults = { ...autocompleteResults, ...recommendationsResults };
-
-  const activeSectionsWithData = getActiveSectionsWithData(activeSections, sectionResults);
-
+  // Add to active sections the results data and refs when autocomplete results or recommendation results fetched
   useEffect(() => {
-    setActiveSections(zeroStateActiveSections ? zeroStateSections : sections);
```

Comment:
ActiveSection equals this expression `zeroStateActiveSections ? zeroStateSections : sections`  that's already defined in the default state and it doesn't change 

Already if zeroStateActiveSections or sections change it gets updated